### DISCUSSION TITLE
Meeting 3 September 2024

### DISCUSSION THREAD

**Discussion by ram1123** [ram1123/nanoAOD_vvVBS#59] [ToDo]
# DY Sample list

**DY 4to10**
```
/DYto2L-2Jets_MLL-4to10_TuneCP5_13p6TeV_amcatnloFXFX-pythia8/Run3Summer22EENanoAODv12-130X_mcRun3_2022_realistic_postEE_v6-v2/NANOAODSIM
/DYto2L-2Jets_MLL-4to10_TuneCP5_13p6TeV_amcatnloFXFX-pythia8/Run3Summer22EENanoAODv12-130X_mcRun3_2022_realistic_postEE_v6_ext1-v2/NANOAODSIM
```

**DY 10to50**

```
/DYto2L-2Jets_MLL-10to50_TuneCP5_13p6TeV_amcatnloFXFX-pythia8/Run3Summer22EENanoAODv12-130X_mcRun3_2022_realistic_postEE_v6-v2/NANOAODSIM
/DYto2L-2Jets_MLL-10to50_TuneCP5_13p6TeV_amcatnloFXFX-pythia8/Run3Summer22EENanoAODv12-130X_mcRun3_2022_realistic_postEE_v6_ext1-v3/NANOAODSIM
```

**DY >50**
```
/DYto2L-2Jets_MLL-50_TuneCP5_13p6TeV_amcatnloFXFX-pythia8/Run3Summer22EENanoAODv12-130X_mcRun3_2022_realistic_postEE_v6-v2/NANOAODSIM
/DYto2L-2Jets_MLL-50_TuneCP5_13p6TeV_amcatnloFXFX-pythia8/Run3Summer22EENanoAODv12-130X_mcRun3_2022_realistic_postEE_v6_ext1-v1/NANOAODSIM
```

# Screenshot from SMP-24-001 AN for the cross-section values

Reference: [SMP-24-001](https://cms.cern.ch/iCMS/analysisadmin/cadilines?line=SMP-24-001&tp=an&id=2758&ancode=SMP-24-001)

<img width="1217" alt="Screenshot 2024-09-03 at 11 31 42" src="https://github.com/user-attachments/assets/bf4c224b-704e-494b-be99-65615e3a80cf">

# How to compute the cross-section from the genXsec analyzer

Instructions on the GEN twiki: [HowToGenXSecAnalyzer](https://twiki.cern.ch/twiki/bin/viewauth/CMS/HowToGenXSecAnalyzer)

# Task list

- [x] Cross-check the cross-section values for the DY sample for 10to50 one and compute for 4-10GeV.
- [x] Plot all the input variables directly from the official nanoAOD to cross-check if all input variables over which we are selecting are fine like SIP, dxy, dx, etc
    - [x] Get this for both DY and data
- [ ] Prepare the cut-flow table for the ZX selections
- [ ] For ZX estimation use all DY samples, i.e., MLL 4-10, 10-50, and > 50. Furthermore, use the **ext** samples too.

@YujiLee301 @vukasinmilosevic

**Comment by YujiLee301** [comment]
**Update on PreEE Samples**

/DYto2L-2Jets_MLL-50_TuneCP5_13p6TeV_amcatnloFXFX-pythia8/Run3Summer22NanoAODv12-130X_mcRun3_2022_realistic_v5-v2/NANOAODSIM
/DYto2L-2Jets_MLL-50_TuneCP5_13p6TeV_amcatnloFXFX-pythia8/Run3Summer22NanoAODv12-130X_mcRun3_2022_realistic_v5_ext1-v1/NANOAODSIM
/DYto2L-2Jets_MLL-10to50_TuneCP5_13p6TeV_amcatnloFXFX-pythia8/Run3Summer22NanoAODv12-130X_mcRun3_2022_realistic_v5-v2/NANOAODSIM
/DYto2L-2Jets_MLL-10to50_TuneCP5_13p6TeV_amcatnloFXFX-pythia8/Run3Summer22NanoAODv12-130X_mcRun3_2022_realistic_v5_ext1-v1/NANOAODSIM

/DYto2L-2Jets_MLL-4to10_TuneCP5_13p6TeV_amcatnloFXFX-pythia8/Run3Summer22NanoAODv12-130X_mcRun3_2022_realistic_v5-v2/NANOAODSIM
(no ext sample on MLL-4to10 Sample)

@ram1123  I think that's all samples that we should run.
